src/app/[lang]/consultation/gecitkale/_components/SqueezeConsultationForm.tsx

```
@@ -0,0 +1,235 @@
+      interestReasonOther: '',
+      investmentTimeline: '',
+    },
+    validationSchema: Yup.object().shape({
+      firstName: Yup.string().required('First name is required'),
+      lastName: Yup.string().required('Last name is required'),
+      email: Yup.string().email('Invalid email address').required('Email is required'),
+      phoneNumber: Yup.string(),
+      interestReason: Yup.string().required('Please select an interest reason'),
+      interestReasonOther: Yup.string().when('interestReason', {
+        is: 'other',
+        then: (schema) => schema.required('Please specify your reason'),
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)